aiida/tools/importexport/dbimport/backends/django.py

```diff
@@ -377,6 +377,12 @@ def _store_entity_data(
     fields_info = reader.metadata.all_fields_info.get(entity_name, {})
     unique_identifier = reader.metadata.unique_identifiers.get(entity_name, None)
 
+    if entity_name == NODE_ENTITY_NAME:
```

Comment:
this should eventually be removed, but it requires (a) changes to the export code and (b) a migration